pkg/test/aws.go

```diff
@@ -1,53 +0,0 @@
-package test
-
-import (
-	"github.com/aws/aws-sdk-go/aws/client"
-	"github.com/aws/aws-sdk-go/service/autoscaling"
-	"github.com/aws/aws-sdk-go/service/autoscaling/autoscalingiface"
-	"github.com/aws/aws-sdk-go/service/ec2"
-	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
-)
-
-// MockAutoscalingService is a mock implementation of a cloud provider interface
-type MockAutoscalingService struct {
-	autoscalingiface.AutoScalingAPI
-	*client.Client
-
-	DescribeAutoScalingGroupsOutput *autoscaling.DescribeAutoScalingGroupsOutput
-	DescribeAutoScalingGroupsErr    error
-
-	SetDesiredCapacityOutput *autoscaling.SetDesiredCapacityOutput
-	SetDesiredCapacityErr    error
-
-	TerminateInstanceInAutoScalingGroupOutput *autoscaling.TerminateInstanceInAutoScalingGroupOutput
-	TerminateInstanceInAutoScalingGroupErr    error
-}
-
-// DescribeAutoScalingGroups mock implementation for MockAutoscalingService
-func (m MockAutoscalingService) DescribeAutoScalingGroups(*autoscaling.DescribeAutoScalingGroupsInput) (*autoscaling.DescribeAutoScalingGroupsOutput, error) {
-	return m.DescribeAutoScalingGroupsOutput, m.DescribeAutoScalingGroupsErr
-}
-
-// SetDesiredCapacity mock implementation for MockAutoscalingService
-func (m MockAutoscalingService) SetDesiredCapacity(*autoscaling.SetDesiredCapacityInput) (*autoscaling.SetDesiredCapacityOutput, error) {
```

Comment:
I notice we've not implemented functions like this in the new mock AWS - do we not need them?

Reply:
I will implement functionality as needed. As I add new features in upcoming PRs I will add the functionality for the mocking to support the appropriate tests.